nginx/README.md

```diff
@@ -1,6 +1,4 @@
-# NGINX check
-
-![NGINX default dashboard][1]
```

Comment:
Interesting - didn't really consider images as part of the links normalization .. can you try including a section at the top with this image?